test/Generator.V3.Tests.Generated/Space.cs

```diff
@@ -30,6 +30,8 @@ public Space()
         public float? SquareFootArea { get; set; }
         [JsonPropertyName("capabilities")]
         public IDictionary<string, bool>? Capabilities { get; set; }
+        [JsonPropertyName("geometry")]
+        public Array? Geometry { get; set; }
```

Comment:
You need to provide a unit test on how this property can be used including: deserialization, iterating, adding or removing elements, etc.